selectors/products.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import type {ProductComponent} from '../types/store/plugins';
+
+import {getCurrentProductId} from '../utils/products';
+
+export function getCurrentProduct(products: ProductComponent[], pathname: string): ProductComponent | undefined {
+    const productID = getCurrentProductId(products, pathname);
+    return products.find((product) => product.id === productID);
+}
```

Comment:
@harshilsharma63 See E2E tests [here](https://mattermost-cypress-report.s3.amazonaws.com/89397-5f06424-ee-ent-webapp-pr-9262-mm-ee-test:5f06424/mochawesome.html) and check if anything is related to this PR. One thing I noticed in the console log is likely thrown from this code block. See screenshot.
<img width="677" alt="Screen Shot 2021-11-23 at 9 29 54 AM" src="https://user-images.githubusercontent.com/5334504/142959110-6cb30e34-ef82-4345-8a3d-40fd8bb07677.png">

Reply:
This is now resolved. Thanks @harshilsharma63 for fixing!